tests/load/sources/sql_database/mssql_source.py

```diff
@@ -0,0 +1,161 @@
+import uuid
+import random
+from datetime import datetime, date, time
+from typing import Any, Dict, List, TypedDict, cast
+from sqlalchemy import Column, DateTime, Integer, MetaData, String, Table, create_engine, func
+from sqlalchemy import schema as sqla_schema
+from sqlalchemy import text
+from sqlalchemy.exc import SQLAlchemyError
+import mimesis
+
+from dlt.common.pendulum import pendulum, timedelta
+from dlt.common.utils import uniq_id
+from dlt.sources.credentials import ConnectionStringCredentials
+
+
+class MSSQLSourceDB:
```

Comment:
I would've probably created a generic source db that uses sqlalchemy and maybe sqlmodel so we can create an example dataset on any database including an abstraction for manipulating rows. We already have more or less the exact same code for postgres. But that is something for the future maybe.

Reply:
right! we can have a few tables with standard types that should work on all source databases. for that we can extract a few standard tests.

tables with specific datatypes could at least have standardized names. but this is significant amount of work. I did a first step by enabling more source databases here.